reader-gtfs/src/main/java/com/graphhopper/reader/gtfs/TripFromLabel.java

```diff
@@ -297,7 +335,7 @@ void finish() {
                     edges(path).map(edgeLabel -> edgeLabel.edgeIteratorState).collect(Collectors.toList()),
                     lineStringFromEdges(path),
                     edges(path).mapToDouble(edgeLabel -> edgeLabel.distance).sum(),
-                    instructions.stream().collect(Collectors.toCollection(() -> new InstructionList(tr))),
```

Comment:
Was there a reason for this?

Reply:
You're right, that was left over from something. I'm merging this line (for a start :-) ) to master, thanks.